apps/golang/install

```diff
@@ -0,0 +1,99 @@
+
+VERSION="1.19.2"
+
+[ -z "$GOROOT" ] && GOROOT="$HOME/.go"
+[ -z "$GOPATH" ] && GOPATH="$HOME/go"
+
+OS="$(uname -s)"
+ARCH="$(uname -m)"
+
+case $OS in
+    "Linux")
+        case $ARCH in
+        "aarch64")
+            ARCH=arm64
+            ;;
+        "armv6" | "armv7l")
+            ARCH=armv6l
+            ;;
+        "armv8")
+            ARCH=arm64
+            ;;
+        esac
+        PLATFORM="linux-$ARCH"
+    ;;
+
+esac
```

Comment:
this won't work. (armhf systems can run arm64 kernels)

if you want to do architecture detection, pi-apps already sets a variable `arch` for you to utilize.
refer to other scripts like the multimc script
https://github.com/Botspot/pi-apps/blob/81ea2d45221300fd8e7a1a827ccad4bb74eabd47/apps/Minecraft%20Java%20MultiMC5/install#L262-L266

Reply:
Okay, I'll try to work on this later today.

Reply:
@theofficialgman I think I fixed it in commit d4ec0bfe615c0ea91c23260caab58e3cd1c792c2